### ISSUE TITLE
Rails 5 - Bundler could not find compatible versions for gem "activerecord"

### ISSUE THREAD

**Issue by silverdr** [rubysherpas/paranoia#349]
When trying to add paranoia to a fresh Rails 5 installation I get the following:

```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    paranoia (~> 2.0) was resolved to 2.0.0, which depends on
      activerecord (~> 4.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activerecord (= 5.0.0)
```

any clues about the cause/resolution/workaround?

**Comment by joshhepworth** [comment]
It looks like Rails 5 support will be part of version 2.2 of paranoia. If you want/need that today you can update your Gemfile to specify `gem 'paranoia', '~> 2.2.0.pre'`. Since this is a pre-release, you'll likely want to test the functionality with a little extra attention.

**Comment by noahmatisoff** [comment]
Should I submit a PR to change the README? It says for Rails 4 or 5, use '~> 2.0'

**Comment by nickrivadeneira** [comment]
PR opened in #351

**Comment by shivabhusal** [comment]
I encountered similar error 

```
 bundle                                                                   6 ↵  2711  15:12:01 
Fetching gem metadata from https://rubygems.org/
Fetching version metadata from https://rubygems.org/
Fetching dependency metadata from https://rubygems.org/
Resolving dependencies....
Bundler could not find compatible versions for gem "activerecord":
  In snapshot (Gemfile.lock):
    activerecord (= 5.0.0.1)

  In Gemfile:
    paranoia (~> 2.0) was resolved to 2.0.0, which depends on
      activerecord (~> 4.0)

    rails_db was resolved to 1.3.3, which depends on
      ransack (<= 1.7.0) was resolved to 1.7.0, which depends on
        polyamorous (~> 1.2) was resolved to 1.3.1, which depends on
          activerecord (>= 3.0)

    rails_db was resolved to 1.3.3, which depends on
      ransack (<= 1.7.0) was resolved to 1.7.0, which depends on
        polyamorous (~> 1.2) was resolved to 1.3.1, which depends on
          activerecord (>= 3.0)

    rails (>= 5.0.0.1, ~> 5.0.0) was resolved to 5.0.0.1, which depends on
      activerecord (= 5.0.0.1)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by radar** [comment]
Use version 2.2 that was released October 20th.
